src/adapters/scene_direction.js

```diff
@@ -65,15 +67,10 @@ export default class SceneDirection {
   }
 
   setMainRoute(routeId) {
-    let mainRoute = null;
     this.routes.forEach((route) => {
-      if (route.id === routeId) {
-        mainRoute = route
-      }
       this.map.setFeatureState({source: `source_${route.id}`, id: 1}, {isActive: route.id === routeId})
     })
-    this.updateMarkers(mainRoute)
```

Comment:
The merge conflict has not been correctly fixed: these lines are part from a previous PR (#174) and should not be removed.

Reply:
Thank you, just the restored the function as it previously was.